src/core/server/http/integration_tests/http_service.test.ts

```diff
@@ -118,7 +118,7 @@ describe('http service', () => {
       afterEach(async () => await root.shutdown());
 
       it('Should support passing request through to the route handler', async () => {
-        const router = new Router('');
```

Comment:
I think this change means we're losing a test case for an empty string resolving to root. I don't know why we'd want that behavior necessarily, and I'm not sure if this breaks anything, but we should have a test for this case as a known behavior, even if it just throws an error as invalid input (my preference).

Reply:
this is a correct input, because the final route is build from Router path and Route path. we have a lot of such tests in 
https://github.com/elastic/kibana/blob/master/src/core/server/http/http_server.test.ts#L634

but there are not focused on this test case and we can add one specific. also we can throw if the final url is Falsy. that works for you?